uportal-war/src/main/java/org/apereo/portal/security/provider/cas/CasAssertionSecurityContext.java

```diff
@@ -31,6 +33,7 @@
 import org.jasig.services.persondir.support.IAdditionalDescriptors;
 import org.springframework.context.ApplicationContext;
 
+import javax.crypto.Cipher;
 
 /**
  * Implementation of the {@link org.apereo.portal.security.provider.cas.ICasSecurityContext} that reads the Assertion
```

Comment:
📜 Maybe add a reference to the markdown documentation from the module doc comment?